packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/OverviewTabPanel.tsx

```diff
@@ -249,3 +338,60 @@ const RemindersCard = ({
     </Card>
   );
 };
+
+const LinkedDocumentsCard = ({ data }: { data: TransformCreditNotes[] }) => {
+  if (!data || data.length === 0) return null;
+
+  return (
+    <Card
+      sx={{ borderRadius: 3, bgcolor: 'background.paper', px: 2 }}
+      variant="outlined"
+    >
+      <Grid container direction="column">
+        {data?.map((item, index) => (
+          <Grid
+            key={item.title}
+            container
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+            sx={{
+              py: 1.5,
+              ...(index
+                ? { borderTop: '1px solid', borderTopColor: 'divider' }
+                : {}),
+              cursor: 'pointer',
+            }}
+            onClick={item.onClick}
+          >
+            <Grid item container direction="column" xs>
+              <Typography
+                variant="body1"
+                fontWeight="bold"
+                sx={{ textTransform: 'capitalize' }}
+              >
+                {item.title}
+              </Typography>
+              <Typography variant="body2">
+                {item.description}{' '}
+                <Link
```

Comment:
For now, we have no interlining in the SDK. So, even if we have, it seems that `href={item.authorTitle}` is not a valid value. Keep it for now as a label.

Reply:
Done

Reply:
Sorry if I didn't make it clear enough: I meant that if we don't have internal navigation, we shouldn't output the element as a link, even with `#` in the URL. Imagine that the customer page uses hash-routing and the user clicks on the link?